hs_app_timeseries/models.py

```diff
@@ -33,58 +44,146 @@ def create(cls, **kwargs):
             if len(set_series_ids) != len(kwargs['series_ids']):
                 raise ValidationError("Duplicate series IDs are found")
 
-        super(TimeSeriesAbstractMetaDataElement, cls).create(**kwargs)
+        element = super(TimeSeriesAbstractMetaDataElement, cls).create(**kwargs)
+        # set the 'is_dirty' field of metadata object to True
+        # if we are creating the element when there is a csv file
+        # note: in case of sqlite file upload we don't create any resource specific metadata
+        # elements - we do only metadata element updates
```

Comment:
It means in case of sqlite file uploaded we do unly update of metadata elements and we  don't create any new resource  specific metadata elements.